app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CustomersController < ApplicationController
+  def index
+    customers = Customer.all
+
+    render(
+      json: customers.as_json(only: [:id, :name, :registered_at, :address, :city, :state, :postal_code, :phone, :account_credit, :movies_checked_out_count]),
+      status: :ok
+    )
+  end
+
+  def show
+    customer = Customer.find_by(id: params[:id])
+    if customer
+      render(
+        json: customer.as_json(only: [:name, :registered_at, :address, :city, :state, :postal_code, :phone, :account_credit, :movies_checked_out_count]),
+        status: :ok
+      )
+    else
+      render json: {nothing: true}, status: :not_found
+    end
```

Comment:
This will literally send back `{ nothing: true }` as a JSON response. Not sure that's what you intended.